chainer/links/connection/parameter.py

```diff
@@ -7,10 +7,6 @@ class Parameter(link.Link):
 
     """Link that just holds a parameter and returns it.
 
-    .. deprecated:: v1.5
```

Comment:
This is a documented API and thus cannot be removed.
https://docs.chainer.org/en/stable/reference/generated/chainer.links.Parameter.html?highlight=Parameter